ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py

```diff
@@ -1036,3 +1036,36 @@ def test_tensorflow_truediv(
         x=x[0],
         y=x[1],
     )
+    
+  
+# argmin
+@handle_cmd_line_args
+@given(
+    dtype_and_x=statistical_dtype_values(function="argmin"),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.tensorflow.argmin"
```

Comment:
Also, I think this should be `fn_name="ivy.functional.frontends.tensorflow.math.argmin"`.